python.sh

```diff
@@ -82,17 +82,18 @@ pushd "$INSTALLROOT/bin"
   [[ -x python3-config ]] || ln -nfs "$PYTHON_CONFIG_BIN" python3-config
 popd
 
+## Disable this for now as xrootd python bindings works only with pip==23.0.1 setuptools==65.5.1
 # Make sure we have the latest pip
-env PATH="$INSTALLROOT/bin:$PATH"                       \
-    LD_LIBRARY_PATH="$INSTALLROOT/lib:$LD_LIBRARY_PATH" \
-    PYTHONHOME="$INSTALLROOT"                           \
-    python3 -m pip install --upgrade pip
+#env PATH="$INSTALLROOT/bin:$PATH"                       \
+#    LD_LIBRARY_PATH="$INSTALLROOT/lib:$LD_LIBRARY_PATH" \
+#    PYTHONHOME="$INSTALLROOT"                           \
+#    python3 -m pip install --upgrade pip
 
 # Install Python SSL certificates right away
 env PATH="$INSTALLROOT/bin:$PATH" \
     LD_LIBRARY_PATH="$INSTALLROOT/lib:$LD_LIBRARY_PATH" \
     PYTHONHOME="$INSTALLROOT" \
-    python3 -m pip install 'certifi==2019.3.9'
+    python3 -m pip install certifi
```

Comment:
I'm not sure this is quite right -- if there's an upstream certifi update and this recipe just specifies certifi without a version, then aliBuild will not rebuild Python (and will not install the new certifi), since the recipe won't have changed.

If we specify a version number, then we can bump that when needed and make aliBuild actually rebuild this package. Of course, feel free to bump this to `certifi==2022.12.7`!

Reply:
done